doc/netplan.md

```diff
@@ -567,9 +567,15 @@ not support GSM modems.
 :    Specify the device ID (as given by the WWAN management service) of the modem to match.
      This can be found using ``mmcli``.
 
+``modem-mtu`` (scalar)
+:    If non-zero, only transmit packets of the specified size or smaller, breaking larger packets up into multiple frames.
+
```

Comment:
why would this be modem-mtu, instead of using the 'mtu' key which is already supported for all interface types? (your latest commit says "according to the spec" - what spec?)